contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -72,7 +73,9 @@ library ValidationLogic {
       supplyCap == 0 ||
         (IAToken(reserveCache.aTokenAddress).scaledTotalSupply().rayMul(
           reserveCache.nextLiquidityIndex
-        ) + amount) <=
+        ) +
```

Comment:
I guess adding up the scaled amounts and multiplying will be the same and save 1 operation. Is there a reason you didn't do it?

Reply:
You are right, makes more sense. Will change